cmd/influxd/run/server_test.go

```diff
@@ -1937,70 +1937,15 @@ func TestServer_Query_Regex(t *testing.T) {
 	}
 }
 
-func TestServer_Query_AggregatesCommon(t *testing.T) {
+func TestServer_Query_Aggregates_Int(t *testing.T) {
 	t.Parallel()
-	s := OpenServer(NewConfig(), "")
+	s := OpenDefaultServer(NewConfig(), "")
 	defer s.Close()
 
-	if err := s.CreateDatabaseAndRetentionPolicy("db0", newRetentionPolicyInfo("rp0", 1, 0)); err != nil {
-		t.Fatal(err)
-	}
-	if err := s.MetaStore.SetDefaultRetentionPolicy("db0", "rp0"); err != nil {
-		t.Fatal(err)
-	}
-
-	writes := []string{
```

Comment:
Yes, they were just broken up because it was difficult to isolate individual tests.